daml-foundations/daml-tools/da-hs-daml-cli/tests/Basic.daml

```diff
@@ -0,0 +1,11 @@
+-- Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+daml 1.2
+
+module Temp where
+
+template Start
+  with
+    test : Party
+  where
+    signatory test
```

Comment:
I'm not sure this template would trigger a wrong edge even with a broken implementation. We need a `fetch` and/or `archive` somewhere.

Reply:
Yes, this one does not. I am planning to use the one nick used it is relatively small. Will add it now.